### ISSUE TITLE
ability to skip paid tests on travis-ci

### ISSUE THREAD

**Issue by tyarkoni** [tyarkoni/pliers#279]
I haven't looked into this, so I don't know if there's an easy solution, but the cost of testing all the API services on every PR/merge is likely to get quite high over time. It's not really a concern for most of the APIs, which are in the $1/1,000 requests ballpark, but some are much more expensive--particularly Indico, which costs $6/1,000 requests. Although our usage will probably come down once we figure out the problem discussed in #272, aggregate costs will continue to rise as we add more APIs.

What we should probably do is add some triggering condition that allows us to only test paid APIs on request, or at least at regular intervals. It would be simple enough to probabilistically trigger the API tests within the travis config (e.g., randomly run them 10% of the time), but it would be nice to have something a little more predictable than that. The best-case scenario would be if there's an easy way to manually trigger only the API tests directly from a GitHub PR.

**Comment by tyarkoni** [comment]
Note that this is partly independent of the issue raised in #237 (and solution in #270), as we'll still need some way to trigger/skip the API tests using the new py.test markers once #270 is merged.
